dashboard/config/upstream-kasan.config

```diff
@@ -5882,7 +5889,6 @@ CONFIG_DNOTIFY=y
 CONFIG_INOTIFY_USER=y
 CONFIG_FANOTIFY=y
 CONFIG_FANOTIFY_ACCESS_PERMISSIONS=y
-CONFIG_MOUNT_NOTIFICATIONS=y
```

Comment:
The config needs to be generated on linux-next, or we will lose this coverage on linux-next.